addusers.py

```diff
@@ -336,7 +337,7 @@ def parse_rows(rows):
     session = session.Session(auth=auth)
     
     openstack = Openstack(session=session, nova_version=nova_version, setpass_url=setpass_url)
-    auth_file = config.get("excelsheet", "auth_file")
+    auth_file = get_absolute_path(config.get("excelsheet", "auth_file"))
     worksheet_key = config.get("excelsheet", "worksheet_key")
```

Comment:
Yes, it's needed because the helpdesk user is actually running the script from a different directory via a more locked-down interface, to prevent them from seeing the admin passwords.  But, it makes the relative paths not work.  I think they would work if it were actually set up to be a python package but it isn't (yet).